src/vellum/workflows/nodes/displayable/bases/search_node.py

```diff
@@ -12,6 +12,7 @@
     SearchWeightsRequest,
 )
 from vellum.core import ApiError, RequestOptions
+from vellum.evaluations.utils.uuid import is_valid_uuid
```

Comment:
[blcoking] Can we move this `uuid.py` file into the new top-level `vellum.utils` directory? We should avoid importing from `evluations/` until there's good reason to.